python/execute_notebooks.py

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env python
+"""This module executes all notebooks. It serves the main purpose to ensure that all can be
+executed and work proper independently."""
+import subprocess as sp
+import glob
+import os
+
+os.chdir(os.environ['PROJECT_ROOT'] + '/notebooks')
+
+for notebook in sorted(glob.glob('*.ipynb')):
+    
+    # TODO: The others are not yet ready for this.
+    if notebook != '00_illustrations.ipynb':
```

Comment:
This only executes one of the notebooks not all of them, What is our Roadmap to including all?

Reply:
Added investigation notebook. Will add other notebooks later.